src/main/resources/operations/component.xml

```diff
@@ -0,0 +1,51 @@
+<component name="operations" type="synapse/template">
+  <subComponents>
+    <component name="executeQuery">
+      <displayName>Execute Query</displayName>
+      <file>executeQuery.xml</file>
+      <description>Execute database query</description>
+    </component>
+    <component name="select">
+      <displayName>Select</displayName>
+      <file>select.xml</file>
+      <description>Select query</description>
+    </component>
+    <component name="insert">
+      <displayName>Insert</displayName>
+      <file>insert.xml</file>
+      <description>Insert query</description>
+    </component>
+    <component name="update">
+      <displayName>Update</displayName>
+      <file>update.xml</file>
+      <description>Update query</description>
+    </component>
+    <component name="delete">
+      <displayName>Delete</displayName>
+      <file>delete.xml</file>
+      <description>Delete query</description>
+    </component>
+    <component name="call">
+      <displayName>Stored Procedure</displayName>
+      <file>call.xml</file>
+      <description>Stored procedure</description>
+    </component>
+
+    <!-- transactions -->
```

Comment:
To give a better UX in the VSCode, we can divide these operations into two sections called "Operations" and "Transactions". We can provide a display name similar to https://github.com/wso2-extensions/mediation-utility-module/blob/8d1dfa6d5c18bb563d256b232927942bbbf6079e/src/main/resources/date/component.xml#L20